src/components/UserInfo.js

```diff
@@ -16,4 +17,8 @@ export default class UserInfo {
     this._name.textContent = data.name;
     this._description.textContent = data.description;
   }
+
+  setUserAvatar(avatar) {
```

Comment:
It's great to have a separate setter for the avatar, since it's controlled by a different form